communitiesuk/funding-service-design-fund-store
# Fs 3581 update round definition to cater optional or mandatory application feedback survey questions

https://dluhcdigital.atlassian.net/browse/FS-3581
https://dluhcdigital.atlassian.net/browse/FS-3582

### Description
Make feedback survey questions to optional on frontend

- Added a new column `feedback_survey_config` (json type) for storing all feedback & survey related config
- Move existing data from the `requires_feedback` column to `feedback_survey_config` with a migration script and delete the column after the data is migrated.
- Tested the migration scripts locally
- Updated the fund-round configs with the added column data
- Disabled the automatic deployments to Dev

linked PR's
https://github.com/communitiesuk/funding-service-design-application-store/pull/235
https://github.com/communitiesuk/funding-service-design-frontend/pull/398

### How to test
- Checkout the branch and restart the fund-store container and observe the column `feedback_survey_config` added in round table automatically with the migrated data from `requires_feedback` column (from your db tool)
- run the load scripts to update the params
